App/authRedirect.js

```diff
@@ -7,19 +7,20 @@ let username = "";
 let accessToken = null;
 
 myMSALObj.handleRedirectPromise()
-    .then(handleResponse)
-    .catch(error => {
-        console.log(error);
-
-        // Check for forgot password error
-        // Learn more about AAD error codes at https://docs.microsoft.com/en-us/azure/active-directory/develop/reference-aadsts-error-codes
-        if (error.errorMessage.indexOf("AADB2C90118") > -1) {
-            try {
-                myMSALObj.loginRedirect(b2cPolicies.authorities.forgotPassword);
-            } catch(err) {
-                console.log(err);
+    .then(response => {
+        if (response) {
+            /**
+             * We need to reject id tokens that were not issued with the default sign-in policy.
```

Comment:
Maybe reword this? You're not exactly rejecting the idToken, just only updating the UI if a user signed-in, correct?